collect_app/src/main/java/org/odk/collect/android/tasks/InstanceUploaderTask.java

```diff
@@ -689,4 +683,24 @@ private static String getFileExtension(String fileName) {
         }
         return fileName.substring(dotIndex + 1).toLowerCase(Locale.ROOT);
     }
+
+    private boolean deleteInstance(String jrFormId) {
```

Comment:
How about `shouldDeleteInstance` or `isAutoDeleteEnabled` so it's clear it's a boolean method? For symmetry with `isFormAutoSendEnabled` above, you could also pass this value in.